File: week-02/day-01/draw_chess_table.py
# Crate a program that draws a chess table like this
#
# % % % % 
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % % 
#  % % % %
#
p = ""
for i in range(1,9):
    if i % 2 == 0:
        for j in range(1,9):
            if j % 2 == 0:
                p += "%"
            else:
                p += " "
    else:
        for j in range(1,9):
            if j % 2 == 0:
                p += " "
            else:
                p += "%"
    print(p)
    p = ""

File: week-08/day-01/35_draw_chess_table/draw_chess_table.js
// Crate a program that draws a chess table like this
//
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % % 
//  % % % %
//
var line ="";
for(i=1;i<=8;i++){
    for(j=1;j<=8;j++){
        if((i%2===0 && j%2===0) || (i%2===1 && j%2===1)){
            line += "%";
        }else{
            line += " ";
        }
    }
    console.log(line);
    line ="";
}